### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by m-talbi**
- Added action to handle adding comments
- Added action to handle adding likes
- Fixed bug when incrementing counters
- Added view to handle adding  a new post
- Added more CSS styles

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -5,5 +5,56 @@ def index
...
+
+    if @post.save
+      @post.increment_posts_counter
+      redirect_to "/users/#{user.id}/posts"
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def create_comment
```
Comment (t-yanick) [inline app/controllers/posts_controller.rb]: - [x] Great job working on this milestone. To create a comment on behalf of the `current_user`, you need to have a `comments_controller` just as you had for `posts_controller`
`A comment is created on behalf of the currently logged in user (current_user) of the app.
The comments controller should have new and create actions`

File: config/routes.rb [hunk 2]
```diff
@@ -4,7 +4,11 @@
   # Defines the root path route ("/")
   root "users#index"
   get "/users/:user_id/posts", to: "posts#index", as: "user_posts"
+  get "/users/:user_id/posts/create", to: "posts#new", as: "new_post"
+  post "/users/:user_id/posts/create", to: "posts#create", as: "create_post"
   get "/users/:user_id/posts/:post_id", to: "posts#show", as: "user_post"
+  post "/users/:user_id/posts/:post_id", to: "posts#create_comment", as: "create_comment"
```
Comment (t-yanick) [inline config/routes.rb]: - [x] When you create the `comments_controller`, make sure you update your `routes` accordingly.

File: app/controllers/posts_controller.rb [hunk 3]
```diff
@@ -5,5 +5,56 @@ def index
...
+    user = current_user
+    @comment = Comment.new(text: comment_params[:text], post: Post.find_by(id: params[:post_id]), user:)
+    @comment.increment_comments_counter
+    @comment.save
+    redirect_to "/users/#{user.id}/posts/#{params[:post_id]}"
+  end
+
+  def add_like
```
Comment (t-yanick) [inline app/controllers/posts_controller.rb]: - [x] In thesame light, you should have the `likes_controller`. A user should be able to add a like to posts. The likes controller should have a create action

File: app/views/posts/show.erb [hunk 4]
```diff
@@ -15,4 +15,14 @@
       </div>
     <% end %>
   </div>
-</article>
\ No newline at end of file
+</article>
+<section class="add-comment-section">
```
Comment (t-yanick) [inline app/views/posts/show.erb]: - [x] There should be a form in the `views/comments/new.html.erb` file which accepts input for creating comments. At the moment you are missing this file. Kindly add it and put the form in the file

File: config/routes.rb [hunk 5]
```diff
@@ -4,7 +4,11 @@
...
+  get "/users/:user_id/posts/:post_id/like", to: "posts#add_like", as: "add_like"
```
Comment (t-yanick) [inline config/routes.rb]: - [x]  When you create the `likes_controller`, make sure you update your `routes` accordingly.

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @Kweeka1 , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  current_user method created in the correct file
:heavy_check_mark: Descriptive commit messages in the imperative tense
:heavy_check_mark: All linter checks are passing

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @Kweeka1, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the Git Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**